### PR TITLE
Contents review.

### PR DESCRIPTION
**Pull request by olea**

### CODE REVIEW

File: org.lamport.tla.toolbox.metainfo.xml [hunk 1]
```diff
@@ -33,26 +34,28 @@
...
-            The TLA Toolbox is an IDE (integrated development environment) for the TLA+ tools.  You can use it to:
-            <ul>
-            <li>Create and edit your specs, with the locations of parsing errors
-            marked in the modules.</li>
-            <li>Run the PlusCal translator, with the locations of translation
-            errors marked in the PlusCal code.</li>
-            <li>View the pretty-printed versions of your modules.</li>
-            <li>Run the TLC model checker.
+    <p>
+        TLA+ is a high-level language for modeling programs and systems and is useful for eliminating fundamental design errors, which are hard to find and expensive to correct in code.
```
Comment (jesmg) [inline org.lamport.tla.toolbox.metainfo.xml]:
```suggestion
        TLA+ is a high-level language for modeling programs and systems. It is useful for eliminating fundamental design errors, which are hard to find and expensive to correct in code.
```

File: org.lamport.tla.toolbox.metainfo.xml [hunk 2]
```diff
@@ -33,26 +34,28 @@
...
+    </p>
+    <p>
+        The TLA Toolbox is an IDE (integrated development environment) for the TLA+ tools.  Use it to:</p>
+    <ul>
+        <li>Create and edit your specs, with the locations of parsing errors
```
Comment (jesmg) [inline org.lamport.tla.toolbox.metainfo.xml:46]:
```suggestion
        <li>Create and edit your specs, with the locations of parsing errors.
```

File: org.lamport.tla.toolbox.metainfo.xml [hunk 3]
```diff
@@ -33,26 +34,28 @@
...
+            marked in the modules</li>
+        <li>Run the PlusCal translator, with the locations of translation
```
Comment (jesmg) [inline org.lamport.tla.toolbox.metainfo.xml:48]:
```suggestion
        <li>Run the PlusCal translator, with the locations of translation.
```

File: org.lamport.tla.toolbox.metainfo.xml [hunk 4]
```diff
@@ -33,26 +34,28 @@
...
+            errors marked in the PlusCal code</li>
+        <li>View the pretty-printed versions of your modules</li>
```
Comment (jesmg) [inline org.lamport.tla.toolbox.metainfo.xml:50]:
```suggestion
        <li>View the pretty-printed versions of your modules.</li>
```

File: org.lamport.tla.toolbox.metainfo.xml [hunk 5]
```diff
@@ -33,26 +34,28 @@
...
+        <li>Run the TLC model checker.
             The Toolbox allows you to explorer an error trace produced by
             TLC—for example, by evaluating arbitrary formulas at each step
-            in the trace.
-            </li><li> Run the <a href="http://tla.msr-inria.inria.fr/tlaps">TLA+ proof system.</a>
-            </li></ul>
-        </p>
+            in the trace
+        </li>
+        <li> Run the TLA+ proof system</li>
```
Comment (jesmg) [inline org.lamport.tla.toolbox.metainfo.xml:56]:
```suggestion
        <li> Run the TLA+ proof system.</li>
```

File: org.lamport.tla.toolbox.metainfo.xml (hunk 2 above)
Comment (olea) [reply org.lamport.tla.toolbox.metainfo.xml:46]: appstreamcli validate doesn't like final dots in \<li> items.

File: org.lamport.tla.toolbox.metainfo.xml (hunk 3 above)
Comment (olea) [reply org.lamport.tla.toolbox.metainfo.xml:48]: appstreamcli validate doesn't like final dots in \<li> items.

File: org.lamport.tla.toolbox.metainfo.xml (hunk 4 above)
Comment (olea) [reply org.lamport.tla.toolbox.metainfo.xml:50]: appstreamcli validate doesn't like final dots in \<li> items.

File: org.lamport.tla.toolbox.metainfo.xml (hunk 5 above)
Comment (olea) [reply org.lamport.tla.toolbox.metainfo.xml:56]: appstreamcli validate doesn't like final dots in \<li> items.

**Pull request merged**